Doc/deprecations/pending-removal-in-future.rst

```diff
@@ -0,0 +1,147 @@
+Pending Removal in Future Versions
+----------------------------------
+
+The following APIs were will be removed in the future,
```

Comment:
```suggestion
The following APIs will be removed in the future,
```

Reply:
Thanks, fixed at the same time as resolving the conflict.